File: 프로그래머스/0/120830. 양꼬치/양꼬치.java
class Solution {
    public int solution(int n, int k) {
        int a = n * 12000;
        int b = k * 2000;
        int c = (n / 10) * 2000;
    
    
        int answer = a + b - c;
        return answer;
    }
}

File: 프로그래머스/0/120830. 양꼬치/양꼬치.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n, int k) {
    int foodPrice= n * 12000;
    int drinkPrice = (k*2000) - ((n/10)*2000);
    int answer = foodPrice + drinkPrice;
    return answer;
}